src/devtools/index.js

```diff
@@ -81,7 +81,7 @@ function initApp (shell) {
     })
 
     bridge.on('instance-details', details => {
-      store.commit('components/RECEIVE_INSTANCE_DETAILS', parse(details))
+      store.commit('components/RECEIVE_INSTANCE_DETAILS', parse(details, true))
```

Comment:
Objects, Arrays, Numbers, Strings, and null are already revived on master. Is there any reason we don't want to do this for Sets, Maps, INF, undefined, and NaN?

Reply:
We only revive JSON-compatible data (which means no 'reviving', just like a standard `JSON.parse()`) because we may need to send it back again (state edition and Vuex time traveling).

Reply:
Also, the State edition system expects JSON-compatible data to work.

Reply:
Ah, that makes sense. I've moved the set reconstruction logic into the component, so the store should only contain JSON-compatible data now.